### ISSUE TITLE
PySPARQL Anything Entry Point/CLI

### ISSUE THREAD

**Issue by MarcoR1791** [SPARQL-Anything/PySPARQL-Anything#13]
The title explains it.

Experimenting with the Python build tools I noticed that we can specify an entry point to the library. This in theory would allow the user to for example be able to call the ```SparqlAnything.run``` method directly from the command line without running the Python shell, import statements and create an object.

@enridaga Would you like me to look into this further?

**Comment by enridaga** [comment]
Why not, what would be the advantages compared to the java cli?

**Comment by MarcoR1791** [comment]
One would be that one should be able to do something like
```powershell
$ sparql-anything -q query
```
rather than
```powershell
java -jar sparql.anything-null  -q query
```
A second advantage would be that if one downloads the Python library, then they have access to the CLI, the Python library and possibly also the fuseki server with one download.

**Comment by MarcoR1791** [comment]
I've uploaded a little test in tests/cli.py (commit 3f2bc8c).

It passes all the usual tests. 

Does require 0.9.0.1 to run though.

**Comment by MarcoR1791** [comment]
Solved in commit 5053c94
